src/test/java/ParserTest.java

```diff
@@ -0,0 +1,29 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import duke.DukeException;
+import duke.Parser;
+import duke.command.Command;
+import duke.command.ExitCommand;
+import duke.command.ListCommand;
+
+public class ParserTest {
+    @Test
+    @DisplayName("Check parse input for bye and list")
+    public void checkParseInputByeAndList() {
```

Comment:
Should this be named according to the three-part format (perhaps `parseInput_bye_list_returnsSingleton`?) This also applies to the other tests.